core/import_data/import_records_section_E.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import pandas as pd
+import numpy as np
+from django.db import transaction
+from django.conf import settings
+
+from core.import_data.utils import (
+    check_headers,
+    delete_old_data,
+    get_cp_report_by_country_name,
+)
+from core.models import CountryProgrammeSectionERecord, Substance
+
+
+REQUIRED_COLUMNS = [
+    "Country",
+    "Year",
+]
+
+REPORT_COLUMNS = {
+    "facility": "Facility name or identifier",
+    "total": "Total amount generated",
+    "all_uses": "Amount generated and captured - For all uses",
+    "feedstock_gc": "Amount generated and captured - For feedstock use in your country",
+    "destruction": "Amount generated and captured - For destruction",
+    "feedstock_wpc": "Amount used for feedstock without prior capture",
+    "destruction_wpc": "Amount destroyed without prior capture",
+    "generated_emissions": "Amount of generated emissions",
+}
+
+
+logger = logging.getLogger(__name__)
+FILE_NAME = "SectionCDE.xlsx"
+
+
+def parse_sheet(df):
+    """
+    parse the sheet and import the data in database
+    @param df = pandas dataframe
+    """
+    if not check_headers(df, REQUIRED_COLUMNS, logger):
+        logger.error("Couldn't parse this sheet")
+        return
+
+    # we have a single substance for the whole sheet
+    substance = Substance.objects.filter(name="HFC-23").first()
+    if not substance:
+        logger.warning(
+            "⚠️ Substance 'HFC-23 not found. Please run `./manage.py import_resources` command before this one."
+        )
```

Comment:
Hmm cred ca poti returna None in caz ca nu gasesti HFC-23 si sa opresti import-ul.
Sau poti modifica putin warning-ul ca sa stie si user-ul ca s-au importat entry-urile fara substanta

Reply:
nu prea are cum sa existe un record fara substanta, asa ca am dat return. my bad, eram sigura ca am facut deja asta 🙃